### DISCUSSION TITLE
Train with coco pre-trained weights on custom black and white dataset gives poor results

### DISCUSSION THREAD

**Discussion by FrancescoMandru** [facebookresearch/detectron2#3295] [Q&A]
If I use pre-trained coco weights on a BW dataset could it be a problem? I'm training on a very small dataset (60 samples) labelled with labelme and converted in COCO format as you suggest in the colab example:

```python
# if your dataset is in COCO format, this cell can be replaced by the following three lines:
# from detectron2.data.datasets import register_coco_instances
# register_coco_instances("my_dataset_train", {}, "json_annotation_train.json", "path/to/image/dir")
# register_coco_instances("my_dataset_val", {}, "json_annotation_val.json", "path/to/image/dir")
```

but the problem is that the network is not able to overfit on this small sample giving very bad results. My task is istance segmentation. I didn't found a way to train detectron 2 from scratch for this specific task

**Comment by ppwwyyxx** [comment]
Training on BW dataset would not be a problem

**Reply by forkcc** [reply]
I have the same problem
If I set up a pre-trained model = COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml
Although the AP score is high, that model cannot detect any targets
If I don’t use the pre-trained model, everything is back to normal
I don't understand what is going wrong

**Reply by forkcc** [reply]
My data set has 150 pictures, all pictures are processed in gray

**Comment by Kevin-Delnoije** [comment]
Did you unfreeze the first layers? I think the default cfg freezes first two layers which contain features for colours
Should be somewhere under cfg.model

**Comment by ExtReMLapin** [comment]
Did you manage what you wanted to do ?

On my end i lowered the learning rate by a lot, it w-work-ish but I still have a poor score.
